src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -1364,6 +1364,8 @@ protected void addTranslations()
 		addItemTooltip(MSItems.GUTTER_BALL, "A hardened crystal in the shape of grist");
 		addItemExtra(MSItems.GUTTER_BALL, "press_shift", "Press §eSHIFT§r for more info");
 		addItemExtra(MSItems.GUTTER_BALL, "desc", "Carved from a common ore, it is believed by the natives to be a kind of food to the yet-born deities.");
+		addItem(MSItems.UNION_BUSTER, "Union Buster");
+		addItemTooltip(MSItems.UNION_BUSTER, "Does quite a lot of damage, both societal AND material!");
```

Comment:
Each weapon category has a region of this file it occupies, where you should put this is between what is currently line 673 and 674